pkg/cmd/server/origin/master.go

```diff
@@ -251,6 +251,7 @@ func (c *MasterConfig) Run(kubeAPIServerConfig *kubeapiserver.Config, controller
 	}
 
 	// add post-start hooks
+	aggregatedAPIServer.GenericAPIServer.AddPostStartHookOrDie("node.openshift.io-sharednamespace", c.ensureOpenShiftNodeNamespace)
```

Comment:
after https://github.com/openshift/origin/pull/16517 this won't be needed.

Reply:
Why?

Reply:
Because we auto init namespaces in the role bindings?

Reply:
Switched to the shared mode.

Why is shared resource namespaces still its own hook?

Reply:
> Why is shared resource namespaces still its own hook?

cruft